boot/zephyr/Kconfig

```diff
@@ -27,6 +30,17 @@ partition=MCUBOOT_PAD
 partition-size=0x200
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
+config PM_PARTITION_SIZE_MCUBOOT
+    hex "Flash space reserved for the MCUboot image"
+    default 0xc000 if !BOOT_MIN_PARTITION_SIZE
+    default FPROTECT_BLOCK_SIZE if SOC_NRF9160
+    # nRF5340 can't shrink image into a single block without external crypto
```

Comment:
What block? Maybe we should explain that the SPU is used for locking the flash, and that the smallest lockable unit is 16kb?